docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -0,0 +1,333 @@
+The following documentation and samples are for creating a container-based database on a persistent volume.
+
+##### Running the database inside Kubernetes
+
+Follow these instructions for a basic database setup inside Kubernetes that uses PV (persistent volume) and PVC (persistent volume claim) to persist the data. For more details about database setup and configuration, refer this [page](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/fmw-infra/#running-the-database-inside-kubernetes).
+
+Pull the database image:
+
+```bash
+$ docker pull container-registry.oracle.com/database/enterprise:12.2.0.1
+$ docker tag  container-registry.oracle.com/database/enterprise:12.2.0.1  oracle/database:12.2.0.1
+```
+Create the PV and PVC for the database:
+
+```bash
+$ mkdir /scratch/DockerVolume/DB
+$ chmod 777 /scratch/DockerVolume/DB
+```
+Update the appropriate host/storage path in `db-pv.yaml`.
+
+Create the PV/PVC using these commands:
+
+```bash
+$ cd  weblogic-kubernetes-operator/kubernetes/samples/scripts/create-database-pv-pvc
+$ kubectl create -f db-pv.yaml
+$ kubectl create -f db-pvc.yaml
+```
+Bring up the database and database service using the following commands:
+
+```bash
+$ cd soa-kubernetes-operator/kubernetes/samples/scripts/create-soa-domain
+$ kubectl create -f db-with-pv.yaml
+```
+Verify that the database service status is Ready.
+
+```bash
+$ kubectl get pods,svc -n soans |grep soadb
+po/soadb-0   1/1       Running   0          6m
+
+svc/soadb   ClusterIP   None         <none>        1521/TCP,5500/TCP   7m
+$
+```
+
+#### Running the Repository Creation Utility to set up your database schema
+
+If you want to run RCU from a pod inside the Kubernetes cluster, you can use the Docker
+image that you built earlier as a "service" pod to run RCU.  To do this, start up a
+pod using that image as follows:
+
+```bash
+kubectl run rcu --generator=run-pod/v1 --image oracle/soa:12.2.1.3 -n soans  -- sleep infinity
+```
+
+This will create a Kubernetes deployment called `rcu` containing a pod running a container
+created from the `oracle/soa:12.2.1.3` image which will just run
+`sleep infinity`, which essentially creates a pod that we can "exec" into and use to run whatever
+commands we need to run.
+
+To get inside this container and run commands, use this command:
+
+```bash
+kubectl exec -n soans -ti rcu /bin/bash
+```
+
+When you are finished with this pod, you can remove it with this command:
+
+```bash
+kubectl delete pod rcu
+```
+
+{{% notice note %}}
+You can use the same approach to get a temporary pod to run other utilities
+like WLST.
+{{% /notice %}}
+
+##### Creating schemas
+
+Inside this pod, you can use the following command to run RCU in command-line (no GUI) mode
+to create your schemas required for SOA domains.  You will need to provide the right prefix and connect string.
+You will be prompted to enter the password for the `sys` user, and then the password to use
+for the regular schema users:
```

Comment:
Please add below note here:
NOTE: In case if the ESS application is being deployed in to the soa domain cluster (i.e. for domain types soaess and soaessosb), the user needs to append the components list with "-component ESS" for createRepository and dropRepository RCU commands.